src/ResourceManager/ServiceBus/Commands.ServiceBus/Cmdlets/Subscription/NewAzureServiceBusSubscription.cs

```diff
@@ -58,7 +58,12 @@ public class NewAzureRmServiceBusSubscription : AzureServiceBusCmdletBase
         [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "Dead Lettering On Message Expiration")]
         [ValidateSet("TRUE", "FALSE", IgnoreCase = true)]
         [ValidateNotNullOrEmpty]
-        public bool? DeadLetteringOnMessageExpiration { get; set; }        
+        public bool? DeadLetteringOnMessageExpiration { get; set; }
+
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "Value that indicates whether a subscription has dead letter support on filter evaluation exceptions.")]
+        [ValidateSet("TRUE", "FALSE", IgnoreCase = true)]
+        [ValidateNotNullOrEmpty]
```

Comment:
@v-Ajnava same comment about removing `ValueFromPipelineByPropertyName`, `ValidateSet` and `ValidateNotNullOrEmpty`

Reply:
Resolved